dist/yall.js

```diff
@@ -60,16 +61,21 @@ function yall (options) {
       if (dataAttrs[dataAttrIndex] in element.dataset) {
         win["requestAnimationFrame"](() => {
           element.setAttribute(dataAttrs[dataAttrIndex], element.dataset[dataAttrs[dataAttrIndex]]);
+          element.classList.remove(lazyClass);
```

Comment:
I moved this oppertation here so that the class gets removed regardless of weather or not intersection observer is being used.